README.md

```diff
@@ -239,12 +246,17 @@ For other formulations see the documentation for
 ``` r
 estimates <- epinow(
   reported_cases = reported_cases,
-  generation_time = generation_time_opts(generation_time),
-  delays = delay_opts(incubation_period + reporting_delay),
+  generation_time = generation_time_opts(example_generation_time),
+  delays = delay_opts(example_incubation_period + reporting_delay),
   rt = rt_opts(prior = list(mean = 2, sd = 0.2)),
   stan = stan_opts(cores = 4, control = list(adapt_delta = 0.99)),
   verbose = interactive()
 )
+#> WARN [2023-10-23 09:19:28] epinow: There were 92 divergent transitions after warmup. See
```

Comment:
Interesting. Will see what happens in the next automatic README update and if problematic revert to certain (just thought it would be good to show the full argument list in the example).

Reply:
yes agree it would so this is not ideal!